src/pause.sol

```diff
@@ -51,7 +83,7 @@ contract DSPause is DSAuth, DSNote {
 
     // --- init ---
 
-    constructor(uint delay_, address owner_, DSAuthority authority_) public {
+    constructor(uint delay_, address owner_, address authority_) public {
```

Comment:
Is it possible that this change will break other repos if they import this into their lib?

Reply:
Everything else LGTM

Reply:
Yes might break them, but that can be updated. I actually prefer to have any param type (and return values) as pure `address`, forcing them to specific interfaces makes sometimes the composability annoying.